CODEOWNERS

```diff
@@ -0,0 +1,10 @@
+/.github @hiddenmarten @AlexGluck
+/api @sircthulhu @Kirill-Garbar
+/charts @hiddenmarten @AlexGluck 
+/cmd @sircthulhu @Kirill-Garbar
+/config @sircthulhu @Kirill-Garbar
+/hack @sircthulhu @Kirill-Garbar
+/internal @sircthulhu @Kirill-Garbar
+/test @hiddenmarten @AlexGluck
```

Comment:
I think we should add `@sircthulhu @Kirill-Garbar` to `/test` directory as Golang implementation maintainers should be responsible for e2e tests too